src/Aspire.Dashboard/Components/Pages/Resources.razor

```diff
@@ -57,21 +58,25 @@
             }
             <FluentDataGrid Virtualize="true" GenerateHeader="GenerateHeaderOption.Sticky" ItemSize="46" Items="@FilteredResources" ResizableColumns="true" GridTemplateColumns="@gridTemplateColumns" RowClass="GetRowClass" Loading="_isLoading">
                 <ChildContent>
-                    <PropertyColumn Title="@Loc[nameof(Dashboard.Resources.Resources.ResourcesTypeColumnHeader)]" Property="@(c => c.ResourceType)" Sortable="true" />
-                    <TemplateColumn Title="@ControlsStringsLoc[nameof(ControlsStrings.NameColumnHeader)]" Sortable="true" SortBy="@_nameSort">
+                    <PropertyColumn Title="@Loc[nameof(Dashboard.Resources.Resources.ResourcesTypeColumnHeader)]" Property="@(c => c.ResourceType)" Sortable="true" Tooltip="true" TooltipText="@(c => c.ResourceType)"/>
```

Comment:
Does this need to be localized?

Reply:
It's not currently being localized and comes from DCP, so I would say no. We don't know all the types that we might be receiving

Reply:
Ok will confirm if they are keywords - we may want to skip localization

Reply:
If they do need localization, it should happen in a follow-up

Reply:
IMO we should be localising resource types and statuses, either with display strings coming from DCP, or with lookups on the dashboard side when the set is fixed.

Reply:
It seems like that's something that should be coming from DCP;otherwise we will have to keep a parallel list of available resource types.